webapi/src/test/scala/org/knora/webapi/messages/util/rdf/JsonLDObjectSpec.scala

```diff
@@ -624,11 +624,6 @@ object JsonLDObjectSpec extends ZIOSpecDefault {
   )
 
   private def smartIriValueSuiteGivenEmptyMap = suite("when given an empty map")(
-    test("requireTypeAsKnoraApiV2ComplexTypeIri should fail with a BadRequestException") {
```

Comment:
Why tests in this file were removed?

Reply:
Because `requireTypeAsKnoraApiV2ComplexTypeIri` was removed. What is not there cannot be tested.